src/renderer/themes/lens-dark.ts

```diff
@@ -45,8 +45,6 @@ const lensDarkTheme: LensTheme = {
     "tableBgcStripe": "#2a2d33",
     "tableBgcSelected": "#383c42",
     "tableHeaderBackground": "#262b2f",
-    "tableHeaderBorderWidth": "1px",
-    "tableHeaderBorderColor": "#36393e",
```

Comment:
@aleksfront I'm concerned here you just eliminated 2 theme CSS vars, which would technically be a breaking change for any extension builders like us out there who might be using these variables for whatever purpose. While you don't need them in the TableHead component anymore, I don't think it's right to just remove them from theme altogether, at least not until Lens 7, assuming Lens is still following semantic versioning rules for the sake of extension developers...

Reply:
I believe usage of these variables might be extremely rare. However, I agree this might be considered as a breaking change and we should restore them.